packages/aws-cdk-lib/aws-rds/lib/serverless-cluster.ts

```diff
@@ -445,19 +458,34 @@ abstract class ServerlessClusterNew extends ServerlessClusterBase {
     const minCapacity = options.minCapacity;
     const maxCapacity = options.maxCapacity;
 
+    const timeoutAction = options.timeoutAction;
+
     if (minCapacity && maxCapacity && minCapacity > maxCapacity) {
       throw new Error('maximum capacity must be greater than or equal to minimum capacity.');
     }
 
-    const secondsToAutoPause = options.autoPause?.toSeconds();
-    if (secondsToAutoPause && (secondsToAutoPause < 300 || secondsToAutoPause > 86400)) {
-      throw new Error('auto pause time must be between 5 minutes and 1 day.');
-    }
+    const secondsBeforeTimeout = options.secondsBeforeTimeout.toSeconds();
+   if  (secondsBeforeTimeout && (secondsBeforeTimeout < 60 || secondsBeforeTimeout > 600)) {
+     throw new Error('seconds before timeout must be between 60 and 600 seconds.');
+   }
+
+   if (timeoutAction && (timeoutAction !== 'ForceApplyCapacityChange' && timeoutAction !== 'RollbackCapacityChange')) {
+     throw new Error('timeout action must be "ForceApplyCapacityChange" or "RollbackCapacityChange".');
+  }
+
+   const secondsToAutoPause = options.autoPause?.toSeconds();
+   if (secondsToAutoPause && (secondsToAutoPause < 300 || secondsToAutoPause > 86400)) {
+     throw new Error('auto pause time must be between 5 minutes and 1 day.');
+  }
 
     return {
       autoPause: (secondsToAutoPause === 0) ? false : true,
       minCapacity: options.minCapacity,
       maxCapacity: options.maxCapacity,
+
+      secondsBeforeTimeout: (secondsBeforeTimeout == 0) ? undefined : secondsBeforeTimeout,
```

Comment:
Shouldn't the default be `300`, not `0`, as [specified in the doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-dbcluster-scalingconfiguration.html#cfn-rds-dbcluster-scalingconfiguration-secondsbeforetimeout) ? Plus `0` would fall outside of the allowed range (60 to 600).

Reply:
I believe these are not required, so we need to check if it exists and for validity. I can alter that if I'm mistaken.

Reply:
One thing I forgot to mention : it should probably be `===` instead of `==` if you're keeping this code, to be consistent with the other ternary operators in this block.